tests/common/plugins/tacacs.py

```diff
@@ -14,9 +14,15 @@ def configure_tacacs(ptfhost, duthost, creds, tacacs_server_ip):
     config_facts  = duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
     for tacacs_server in config_facts.get('TACPLUS_SERVER', {}):
         duthost.shell("sudo config tacacs delete %s" % tacacs_server)
+
     duthost.shell("sudo config tacacs add %s" % tacacs_server_ip)
     duthost.shell("sudo config tacacs authtype login")
 
+    if config_tacacs_on_switch:
+        duthost.shell("sudo config tacacs add %s" % creds['tacacs_servers'][0])
+        duthost.shell("sudo config tacacs authtype pap")
+        duthost.shell("sudo useradd %s --password 111111" % creds['tacacs_ro_user'], module_ignore_errors=True)
```

Comment:
The password is a different password than the one configured in creds['tacacs_ro_user_passwd'].
I can change it to be provided by the user - the same as config_tacacs_on_switch parameter.

Reply:
If config_tacacs_on_switch flag is set to true, then the existing tacacs test will not be executed.
Should config_tacacs_on_switch be made another test ?

This code is invoked by both test_ro_user and test_rw_user, here the username used is test_ro_user. Also, how is the ssh tested after "useradd"?

Reply:
@SuvarnaMeenakshi 
the original test will be executed, but the tacacs server will be different. 

I added an option to choose the username and password to the local user being added. (by default it will be tacacs_ro_user, and 111111 as password). 

ssh is being tested after "useradd" since it's a different password than the one configured for tacacs.

Reply:
If the idea is just to use remote server IP, then probably password/username etc also should be taken from lab.yml instead of hard coding strings here. only the server IP should be modified as remote server or  ptf-docker IP address will be used by default.
If we are just testing remote server, why is the auth type changed to pap?